packages/toolpad-app/src/types.ts

```diff
@@ -165,3 +165,13 @@ export interface AppTheme {
 }
 
 export type VersionOrPreview = 'preview' | number;
+
+export interface GithubRelease {
+  tag_name: string;
+  html_url: string;
+}
+
+export interface GithubReleaseCache {
```

Comment:
I don't believe there's a need to have types like this shareable. Interfaces that are local to an implementation can stay there. (In this case, it's probably not even needed)